mods/cnc/maps/shellmap/map.yaml

```diff
@@ -994,6 +994,9 @@ Rules:
 			Effect: Desaturated
 		LuaScript:
 			Scripts: shellmap.lua
+		MusicPlaylist:
+			StartingMusic: map1
```

Comment:
You're leaking state somewhere.  If you start TD without scores.mix it will play map1.  Go Extras &rarr; Music &rarr; Install Music &rarr; Back and it will sometimes play the score screen music.